starter_code/public/stylesheets/styles.css

```diff
@@ -5,4 +5,159 @@ dark-blue: #0A2C42;
 grey: rgba(145, 145, 145, 0.481);
 font-family: Arial, Helvetica, sans-serif
 */
+
```

Comment:
Good that you used the boilerplate, but if there are things you don't use, it's better if you delete them.

Reply:
Gotcha 🧐